### DISCUSSION TITLE
Grouping of switches from each device

### DISCUSSION THREAD

**Discussion by tommyzacek** [merdok/homebridge-miot#401] [Ideas]
Would it be possible to add an option to group switches on a given device? For example, the "Dyson Pure Cool" plugin will create a main device that is controlled in one tile, such as turning on the air purifier and its fan speed. And a second tile will appear next to it, and after clicking it, a window will open with more switches, for example "night mode", "continuous monitoring", "Jet Focus". Such a classification seems clearer to me. Is it technically possible?

**Comment by merdok** [comment]
Currently no, all accessories which belong to a device are placed in the same group making it directly clear which accessories belong to which device. Without that if you would have multiple devices you would not know to which device the accessories belong to. That would be even worse when you would have multiple devices if the same type...
